js/core/dfn.js

```diff
@@ -21,11 +21,13 @@ define(
                     } else {
                         dfn.attr("data-dfn-for", (dfn.closest("[data-dfn-for]").attr("data-dfn-for") || "").toLowerCase());
                     }
-                    var title = dfn.dfnTitle();
-                    if (!conf.definitionMap[title]) {
-                        conf.definitionMap[title] = [];
-                    }
-                    conf.definitionMap[title].push(dfn);
+                    var titles = dfn.dfnTitle( true );
```

Comment:
Sorry, can you please change the argument so it's not a [Boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html)? Either change it to an object (`{isDefinition: true}`) or a string.